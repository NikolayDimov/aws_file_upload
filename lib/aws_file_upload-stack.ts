import * as cdk from "aws-cdk-lib";
import * as s3 from "aws-cdk-lib/aws-s3";
import { Construct } from "constructs";

export class AwsFileUploadStack extends cdk.Stack {
    constructor(scope: Construct, id: string, props?: cdk.StackProps) {
        super(scope, id, props);

        // S3 Bucket
        const bucket = new s3.Bucket(this, "FileUploadBucketAwsExam");
    }
}
