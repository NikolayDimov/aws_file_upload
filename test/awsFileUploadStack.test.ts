import * as cdk from "aws-cdk-lib";
import { AwsFileUploadStack } from "../lib/aws_file_upload-stack";
import "jest-cdk-snapshot";

describe("AwsFileUploadStack", () => {
    it("should match the CloudFormation snapshot", () => {
        const stack = new cdk.Stack();

        new AwsFileUploadStack(stack, "TestStack");

        expect(stack).toMatchCdkSnapshot();
    });
});
