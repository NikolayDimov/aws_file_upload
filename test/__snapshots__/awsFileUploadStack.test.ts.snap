// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AwsFileUploadStack should match the CloudFormation snapshot 1`] = `{}`;
